source/plugin/commands/arguments.rst

```diff
@@ -174,25 +174,27 @@ The parser in this example reads two input arguments and converts them to a vect
 .. code-block:: java
 
    import com.flowpowered.math.vector.Vector2i;
-   import org.spongepowered.api.util.command.args.ArgumentParseException;
-   import org.spongepowered.api.util.command.args.CommandArgs;
+   import org.spongepowered.api.command.args.ArgumentParseException;
+   import org.spongepowered.api.command.args.CommandArgs;
    import org.spongepowered.api.text.Text;
-   import org.spongepowered.api.util.command.args.CommandElement;
+   import org.spongepowered.api.command.args.CommandElement;  
```

Comment:
Not sure if they have been pruned to match style guides for the docs, but this code-block is missing import statements required for the example to compile.

```

import com.flowpowered.math.vector.Vector2i;
import org.spongepowered.api.command.CommandSource;
import org.spongepowered.api.command.args.ArgumentParseException;
import org.spongepowered.api.command.args.CommandArgs;
import org.spongepowered.api.command.args.CommandContext;
import org.spongepowered.api.command.args.CommandElement;
import org.spongepowered.api.text.Text;
import org.spongepowered.api.text.Texts;

import java.util.Collections;
import java.util.List;
```